diff --git a/lib/app/getLocalFileDetails/index.js b/lib/app/getLocalFileDetails/index.js
index b4080743aa640fd21238616c539085ff1b28a038..6508a5ec0f8112033fa6248e062b442896d2047a 100644
--- a/lib/app/getLocalFileDetails/index.js
+++ b/lib/app/getLocalFileDetails/index.js
@@ -40,7 +40,7 @@ const getLocalFileDetails = ({
           filePath,
           compression
         });
-        const normalizedFilePath = normalizeFilenames ? // remove matched capture groups
+        const normalizedFilePath = normalizeFilenames && normalizeFilenames.test(filePath) ? // remove matched capture groups
         filePath // find all matching segments
         .split(normalizeFilenames).reduce((partiallyNormalizedPath, matchingSegment) => // remove matching segment from normalized path
         partiallyNormalizedPath.replace(matchingSegment, ''), filePath) : filePath;